### PR TITLE
Move zmon related components to visibility namespace

### PR DESCRIPTION
**Pull request by linki**
So that EagleEye can fix it in case of emergency: https://github.bus.zalan.do/teapot/issues/issues/1030

### CODE REVIEW

File: cluster/manifests/visibility/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```
Comment (mikkeloscar) [inline cluster/manifests/visibility/namespace.yaml]: I think you need to name this file `01-namespace.yaml` or something like that, to ensure CLM creates the namespace before creating resources in it. :X

Comment (aermakov-zalando) [reply cluster/manifests/visibility/namespace.yaml]: The directory needs to be renamed as well, otherwise logging-agent change above would fail.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by njuettner** [comment]
👍

**Pull request merged**